source/_integrations/homekit_controller.markdown

```diff
@@ -79,17 +79,82 @@ HomeKit IP accessories for these device types may work with some caveats:
+- via [Apple Thread border router](#adding-a-homekit-device-to-home-assistant-via-apple-thread-border-router)
+
+This section describes how to add it via Home Assistant.
+
+### Prerequisites
+
+- A HomeKit device which supports Thread. This is indicated by the Thread label on the packaging.
+- Make sure HomeKit controller has been [joined using Bluetooth](#adding-a-homekit-device-through-bluetooth). 
+- **Thread network**: In order to use HomeKit over Thread, you need a working border router. 
+    - Make sure your Home Assistant device is on the same network (LAN) as the border router.
+    - If you have a Home Assistant Yellow or SkyConnect, you can enable multiprotocol to set up an Open Thread border router and with that a Thread network. 
+      - Documentation on [enabling multiprotocol on Yellow](https://yellow.home-assistant.io/guides/enable-multiprotocol/)
+      - Documentation on [enabling multiprotocol on SkyConnect](https://skyconnect.home-assistant.io/procedures/enable-multiprotocol/)
+
+### To add a HomeKit  device to a Thread network via Home Assistant
+
+1. To open the device configuration page, on the **HomeKit** integration, select the **device**.
+1. Under **Diagnostic**, you can see the **Thread Status** as **Disabled**.
+    ![Device configuration page](/images/integrations/homekit_controller/homekit_controller_add_02.png)
+1. To enable Thread, under **Configuration**, select **Press**. This will provision the preferred Thread credentials.
+   - The status has now changed: 
+     - Depending on the device type, the mesh size and health, the Thread status can be **Child**, **Router**, or **Leader**. 
+     ![Thread status](/images/integrations/homekit_controller/homekit_controller_add_02.png)
+   - That's it. Your HomeKit device now communicates via Thread.
+
+## Adding a HomeKit device to Home Assistant via Apple Thread border router
+
+There are two methods to add a HomeKit device to a Thread network:
+
+- via [Home Assistant](#adding-a-homekit-device-to-a-thread-network-via-home-assistant)
+- via Apple Thread border router
+
+This section describes how to add a HomeKit device using an Apple Thread border router device such as a HomePod mini.
+
+### Prerequisites
+
+- An Apple device that can act as a Thread border router, such as a HomePod mini.
+- A HomeKit device which supports Thread. This is indicated by the Thread label on the packaging.
+- Make sure your Home Assistant instance is on the same network (LAN) as the border router.
+- Make sure the HomeKit device has been paired in the Apple Home app (using an iOS app).
+
+### To add a HomeKit device to Home Assistant via Apple Thread border router
+
+1. Unpair the HomeKit device from the Apple Home app. Don't reset the device.
+   - This leaves the Thread network details on the HomeKit device.
+1. The device will be automatically discovered by the HomeKit controller integration in Home Assistant. 
+   - It will appear as a discovered device over Thread.
```

Comment:
Just like Bluetooth, you'll have to click configure here. Then it will be added.

Reply:
Added missing steps.